src/runtime/tests/refiner-test.ts

```diff
@@ -583,7 +583,7 @@ describe('KTExtracter', () => {
       const schema = manifest.particles[0].handleConnectionMap.get('input').type.getEntitySchema();
       const query: string = KTExtracter.fromSchema(schema, escaper);
       assert.strictEqual(query,
-        'val a = data.a as Double\nval b = data.b as Double\nreturn ((b + (a * 3)) > 300) && ((a > 3) && (!(a == 100))) && ((b > 20) && (b < 100))');
+        'val a = data.singletons["a"] as Double\nval b = data.singletons["b"] as Double\n((b + (a * 3)) > 300) && ((a > 3) && (!(a == 100))) && ((b > 20) && (b < 100))');
```

Comment:
It might be easier to read if it's formatted like: 
```
`\
val a = data.singletons["a"] as Double
val b = ...
`
```

Reply:
Thanks